### DISCUSSION TITLE
Prevent unauthorized requests to the server

### DISCUSSION THREAD

**Discussion by MooshiMochi** [FlareSolverr/FlareSolverr#1061] [Q&A]
I used docker to run an instance of flaresolverr on my VPS, and noticed that there's nothing stopping me from making a request from a different machine to the VPS's IP.
Is there some sort of configuration to make restrict access to it? Perhaps set an API key as an environment variable?

**Comment by ilike2burnthing** [comment]
Don't expose the port to the internet.

**Comment by MooshiMochi** [comment]
Sure, that's an idea, but you can never be too safe with these things.
I believe an additional layer of security will be of great help in case the port/ip gets leaked by accident.
It could even be as simple as setting an API key as an environment variable when starting up the server.
